mining/src/main/java/net/runelite/client/plugins/mining/MiningOverlay.java

```diff
@@ -72,6 +73,11 @@ private MiningOverlay(final Client client, final MiningPlugin plugin, final Mini
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		if (config.highlightActiveDaeyaltEssence() && plugin.getActiveDaeyaltEssence() != null)
```

Comment:
What's the point in checking if the config is enabled if `plugin.getActiveDaeyaltEssence()` would always be null with the option disabled?